### DISCUSSION TITLE
add option to compute jacobian determinants

### DISCUSSION THREAD

**Discussion by j042** [tataratat/splinepy#256] [General]
det(jacobian) is as interesting as jacobian matrix sometimes and it would be nice to have an option to have them returned together. In case J doesn't have a square shape, we could either return its weight or error.

**Comment by jzwar** [comment]
Related to FFD improvements :
https://github.com/tataratat/gustaf/discussions/108

**Comment by jzwar** [comment]
I think, creating a new function in SplineBase, (pure python) that can compute the geometric_measure would also be a good first task for students. Maybe @FSchwar would be interested? @j042 would you mind coordinating while I am on vacation?

**Reply by j042** [reply]
sure
